operator/src/test/java/oracle/kubernetes/operator/helpers/TuningParametersStub.java

```diff
@@ -39,6 +39,11 @@ public PodTuning getPodTuning() {
         LIVENESS_PERIOD);
   }
 
+  @Override
+  public MainTuning getMainTuning() {
```

Comment:
what do all these values mean?

Reply:
It's a test stub. I had to use values to get it to compile. I think the original code is needlessly confusing, but have had no real reason to fix it.